app/assets/images/payment_icons/vipps.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" width="38" height="24" role="img" aria-labelledby="pi-vipps"><title id="pi-vipps">Vipps</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" fill="#000"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32z" fill="#FF5B24"/><path fill-rule="evenodd" clip-rule="evenodd" d="M21.753 9.294c1.017 0 1.888-.779 1.888-1.897 0-1.12-.871-1.897-1.888-1.897-1.016 0-1.888.778-1.888 1.897 0 1.118.872 1.897 1.888 1.897zm2.47 3.064c-1.26 1.653-2.59 2.796-4.939 2.796-2.396 0-4.26-1.459-5.712-3.599-.581-.875-1.477-1.07-2.13-.608-.606.438-.75 1.362-.194 2.165C13.257 16.2 16.04 18 19.284 18c2.978 0 5.302-1.46 7.117-3.891.677-.9.653-1.824 0-2.335-.606-.487-1.501-.316-2.179.584z" fill="#fff"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-vipps"><title id="pi-vipps">Vipps</title><g fill-rule="evenodd" clip-rule="evenodd" id="pi-vipps-background_vipps"><path fill="#FFF" d="M36 24H2c-1.333 0-2-.7-2-2V2C0 .667.7 0 2 0h34c1.333 0 2 .7 2 2v20c0 1.333-.667 2-2 2z"/><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3" stroke-opacity=".07" d="M36 24c1.333 0 2-.7 2-2V2c0-1.333-.7-2-2-2H2C.7 0 0 .7 0 2v20c0 1.333.7 2 2 2h34z"/></g><path fill-rule="evenodd" clip-rule="evenodd" fill="#FF5B24" d="M25.1 6.4c0 .7-.25 1.3-.75 1.8a2.59 2.59 0 01-1.75.65c-.666 0-1.216-.217-1.649-.65-.5-.5-.75-1.1-.75-1.8 0-.733.25-1.316.75-1.75.433-.433.983-.65 1.649-.65.667 0 1.25.217 1.75.65.5.5.75 1.083.75 1.75zm3.5 5.65c.4.367.6.816.6 1.35.066.533-.134 1.083-.601 1.649-2.533 3.3-5.566 4.95-9.1 4.95-2.067 0-3.983-.566-5.75-1.7-1.733-1-3.25-2.533-4.55-4.6-.333-.467-.467-.967-.4-1.5.067-.5.3-.883.7-1.15.4-.3.867-.383 1.4-.25s.967.467 1.3 1c.933 1.4 1.983 2.5 3.15 3.3 1.3.801 2.683 1.2 4.15 1.2 1.366 0 2.583-.3 3.65-.899.833-.534 1.716-1.4 2.649-2.601.4-.6.867-.933 1.4-1 .535-.132 1.002-.049 1.402.251z" id="pi-vipps-icon_vipps"/></svg>
```

Comment:
Same comments applied here: unused id attributes and border needs to follow other icons.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-vipps"><title id="pi-vipps">Vipps</title><g fill-rule="evenodd" clip-rule="evenodd"><path fill="#FFF" d="M36 24H2c-1.333 0-2-.7-2-2V2C0 .667.7 0 2 0h34c1.333 0 2 .7 2 2v20c0 1.333-.667 2-2 2z"/><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3" stroke-opacity=".07" d="M36 24c1.333 0 2-.7 2-2V2c0-1.333-.7-2-2-2H2C.7 0 0 .7 0 2v20c0 1.333.7 2 2 2h34z"/></g><path fill-rule="evenodd" clip-rule="evenodd" fill="#FF5B24" d="M25.1 6.4c0 .7-.25 1.3-.75 1.8a2.59 2.59 0 01-1.75.65c-.666 0-1.216-.217-1.649-.65-.5-.5-.75-1.1-.75-1.8 0-.733.25-1.316.75-1.75.433-.433.983-.65 1.649-.65.667 0 1.25.217 1.75.65.5.5.75 1.083.75 1.75zm3.5 5.65c.4.367.6.816.6 1.35.066.533-.134 1.083-.601 1.649-2.533 3.3-5.566 4.95-9.1 4.95-2.067 0-3.983-.566-5.75-1.7-1.733-1-3.25-2.533-4.55-4.6-.333-.467-.467-.967-.4-1.5.067-.5.3-.883.7-1.15.4-.3.867-.383 1.4-.25s.967.467 1.3 1c.933 1.4 1.983 2.5 3.15 3.3 1.3.801 2.683 1.2 4.15 1.2 1.366 0 2.583-.3 3.65-.899.833-.534 1.716-1.4 2.649-2.601.4-.6.867-.933 1.4-1 .535-.132 1.002-.049 1.402.251z"/></svg>
```